networkconfig/mainnet.go

```diff
@@ -22,5 +22,8 @@ var Mainnet = NetworkConfig{
 		// Blox's exporter nodes.
 		"LS0tLS1CRUdJTiBSU0EgUFVCTElDIEtFWS0tLS0tCk1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBeSttZUZmOGwyZ2lzK01YU1pXc3oKTFhPTjRodXJ1ak4zQnNSRTJCU1FuV3RMUHY1Y2prSW0xa2FxRThqWEtBbU5nRkwrT3o0eldpczhFRVFhMjB1UgpyM01RM1NMQnlpaWFRYjNDeStjMFg1UDFsTFBBYzVxNnhiZlJBZXp3K2dUUkFYSXo4RXBwaGdVblNyVUQvOXp2CmZ1OFRaQkVLSlYrcnFDRTZZN0FpcU9jVUsrNHF3TWUyeWQrMW9rRld2d2E3c3h4T2VZNGdBcG9jTENNQmRzKzIKQlY2UVR5aVZaT1daQlhFSjdXMllINHBHMWRlMHdMRUZaUnVkcmE1L3RXUzBqSzRRV0Vhc21WeG5LOUpsSWJDdgplZW5vYWt2M1pjamM4WGs1MmRLWGFuNy9TNDNxdHRJT1MvbFdmRDdxSTZvWXp5aXJhZVh2dDdYbXlhODJIa3JZCkd3SURBUUFCCi0tLS0tRU5EIFJTQSBQVUJMSUMgS0VZLS0tLS0K",
 		"LS0tLS1CRUdJTiBSU0EgUFVCTElDIEtFWS0tLS0tCk1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBeTd0dlFKa01nZUx3ekp5c1lZUlgKRWNyTC9PUHBScDkyUWxEdHZNaXQrdXZKbEtSV1B6OFZCbERtV1UvV0UrTkUxa2FyTDVrRFAxazRMMUFUUjJFWgpLNEpMUUp3RW5kNHVBWlB3Rm1UVTEvWmVDN1kvVmlNcmlyMW1pSzRmcXNnTko5UmVWWjAzQ3hpVGNQQjNHNTE0ClhQaklzaUo0eS8wSlB6cmhQckR5Vmt3SnEwWWRnMWpJMUJkbzVaVm15SkZ4eC9lblcwcVUrNG9iaElGZThlUkEKdjUrbS9aa1lUbnNoMklsVk10UjB5TUQwR0I0YWo3MGQ1VVIwMk1yZkhCWXVLOHpnSitXVkN2R0JVTm9ramVFZQpvWVRsYmQwSzAxRWh1MHN1cStjc0FubU8vaTBaaDVHOVM3MU5EVkc4QnBhdVk5cHYvcFlDa3ZqaHNRdGtQTEJKCjd3SURBUUFCCi0tLS0tRU5EIFJTQSBQVUJMSUMgS0VZLS0tLS0K",
+		// HellmanAlert full nodes
```

Comment:
can we please add here a new line above the comment, and also change the text to 'HellmanAlert exporter nodes.' (to be like the Blox one)?

Reply:
<img width="1240" alt="solve_conflicts" src="https://github.com/bloxapp/ssv/assets/32962819/80cc5780-d7e5-4aef-bcaa-8edaaa8feb2b">
like this?